modules/oe_translation_local/tests/src/Functional/LocalTranslationsTest.php

```diff
@@ -603,6 +611,78 @@ public function testLocalTranslationDashboard(): void {
     $this->assertSession()->linkExistsExact('Edit accepted translation');
   }
 
+  /**
+   * Tests that translations can be synced when the original node is updated.
```

Comment:
Tests that translation values are removed upon sync when source ones are removed